Flow.Launcher/Languages/en.xaml

```diff
@@ -60,6 +60,11 @@
     <system:String x:Key="shadowEffectNotAllowed">Shadow effect is not allowed while current theme has blur effect enabled</system:String>
 
     <!--  Setting Plugin  -->
+    <system:String x:Key="searchplugin">Search Plugin</system:String>
+    <system:String x:Key="searchpluginToolTip">Ctrl+F to Search Plugins</system:String>
+    <system:String x:Key="searchplugin_Noresult_Title">No results to display</system:String>
+    <system:String x:Key="searchplugin_Noresult_Subtitle">Try another search term.</system:String>
+    <system:String x:Key="searchplugin_Noresult_Subtitle_Network">Please check your network connection or try another search term.</system:String>
```

Comment:
These different states, I can't seem to hit the 'searchplugin_Noresult_Subtitle', and displaying about network connection is confusing no?
![image](https://user-images.githubusercontent.com/26427004/194983244-98f243c9-b95d-430b-a989-ae696527f334.png)

Reply:
Should it be Plugin Store on load if no results -> 'No plugins can be loaded, please check your network connection.'
When searching in Plugins tab or Plugin Store and no results found -> 'No results found. Please try a different search.'

Reply:
I simply removed network part, and changed text to 'No results found. Please try a different search.'

Since the network connection verification function is not implemented (I remember that Chinese users have left an issue here) I wanted to put in a simple message. Even when it was not connected, the result was zero, so I tried to put a warning message here. I thought it was a good idea because I didn't need a function code. Let's implement and add the network confirmation part in a separate PR.